tests/unit/test_fracture_network.py

```diff
@@ -314,6 +313,79 @@ def test_add_networks_preserve_tags(self):
         known_tags = np.array([tag1, tag1, tag2, tag2])
         self.assertTrue(np.all(together._edges[2] == known_tags))
 
+    def test_triangle_domain_no_fracs(self):
+        line_1 = np.array([[0, 1], [0, 0]])
+        line_2 = np.array([[1, 0], [0, 1]])
+        line_3 = np.array([[0, 0], [1, 0]])
+        lines = [line_1, line_2, line_3]
+        domain = pp.Domain(polytope=lines)
+
+        mesh_size = 0.5
+        mesh_kwargs = {"mesh_size_frac": mesh_size}
+        mdg = pp.create_fracture_network(domain=domain).mesh(mesh_kwargs)
+        sd = mdg.subdomains(dim=2)[0]
+
+        self.assertTrue(np.isclose(sd.num_cells, 7))
```

Comment:
See comments below regarding such acceptance criteria (I somehow placed them on the last test)

Reply:
thanks, now I'm checking something that should be robust with respect to the resulting grid